src/qt/transactionview.h

```diff
@@ -69,6 +70,9 @@ class TransactionView : public QWidget
     QLineEdit *addressWidget;
     QLineEdit *amountWidget;
 
+    QTimer *m_inputDelayPrefix;
```

Comment:
To also drop my opinion here (though I don't care): Use the scheme of the surrounding code to not confuse new developers. New classes or complete rewrites may be different.